src/main/java/io/supertokens/webserver/api/thirdparty/getUsersByEmail/GetUsersByEmailAPI.java

```diff
@@ -46,7 +47,7 @@ public String getPath() {
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException, ServletException {
         try {
-            String email = req.getParameter("email");
+            String email = InputParser.getQueryParamOrThrowError(req, "email", false);
 
             UserInfo[] users = ThirdParty.getUsersByEmail(super.main, email);
```

Comment:
Doesn't this function call complain about `email` being null.. Cause in other APIs, I have done `assert email != null;`. Like in the `SignInAPI` for emailpassword.

Reply:
It doesn't; neither in IDE nor when building